sql_generators/active_users_aggregates_v3/templates/desktop_query.sql

```diff
@@ -98,7 +78,7 @@ GROUP BY
   distribution_id,
   first_seen_year,
   is_default_browser,
-  language_name,
+  locale,
   app_name,
```

Comment:
Do we want to also aggregate by `is_desktop`? It should all work out because we're adding non-qualifying sources to the `app_name`, but I just want to double-check.

Reply:
Yeah I think it works, we can add it in the view.